tests/shardddl3/run.sh

```diff
@@ -610,6 +610,23 @@ function DM_102_CASE() {
     run_sql_source1 "insert into ${shardddl1}.${tb1} values (1,1);"
     run_sql_source2 "alter table ${shardddl1}.${tb1} add column new_col1 int default -1;"
 
+    sleep 1
+    # wait DM receive source2's DDL
+    found=false
+    for ((k=0; k<10; k++)); do
+        content=$($PWD/bin/dmctl.test DEVEL --master-addr=127.0.0.1:$MASTER_PORT query-status test)
+        master2=$(echo $content | sed 's/"masterBinlog":/"masterBinlog":\n/g' | awk -F')' 'FNR==3{print $1}')
+        syncer2=$(echo $content | sed 's/"syncerBinlog":/"syncerBinlog":\n/g' | awk -F')' 'FNR==3{print $1}')
+        if [ "$master2" != "$syncer2" ]; then
+            found=true
```

Comment:
Is there other cause can lead to `"$master2" != "$syncer2"`?

Reply:
I guess no. syncer was initiated at the position where no modification made in upstream, at then `"$master2" == "$syncer2"`. and only modification after that is source2's DDL which we are waiting